test/integration/patients/patient/patient-flow.js

```diff
@@ -429,142 +424,116 @@ context('patient flow page', function() {
   });
 
   specify('add action', function() {
+    const testProgramAction = getProgramAction({
+      attributes: {
+        name: 'Conditional',
+        published_at: testTs(),
+        archived_at: null,
+        behavior: 'conditional',
+        details: '',
+        days_until_due: 0,
+        sequence: 0,
+      },
+      relationships: {
+        owner: getRelationship(null),
```

Comment:
I think this is just `getRelationship()`

Reply:
Updated in https://github.com/RoundingWell/care-ops-frontend/pull/1249/commits/93742de5055d28fe1db71aa6540c4f6fdc142151.